Bookish/Controllers/MemberController.cs

```diff
@@ -29,9 +29,10 @@ public IActionResult AddMember()
     }
     
     
-    public async Task<ActionResult> MemberQuery(Member member)
+    public IActionResult MemberQuery(Member member)
```

Comment:
For clarity, I would put annotations on each method to indicate if they're GET or POST requests (or something else). You can do this by putting e.g. `[HttpGet]` above the name of the method